feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -160,8 +166,29 @@ fun ForYouScreen(
                     else -> floor(maxWidth / 300.dp).toInt().coerceAtLeast(1)
                 }
 
+                // Workaround to call Activity.reportFullyDrawn from Jetpack Compose.
+                // This code should be called when the UI is ready for use
+                // and relates to Time To Full Display.
+                val interestsLoaded =
+                    interestsSelectionState !is ForYouInterestsSelectionUiState.Loading
+                val feedLoaded = feedState !is ForYouFeedUiState.Loading
+
+                if (interestsLoaded && feedLoaded) {
+                    val localView = LocalView.current
+                    // Unit prevents calling it only once on recomposition, but doesn't prevent when the composable goes out of scope.
```

Comment:
Comment is worded a bit strange. I think you mean something like "We use Unit to call reportFullyDrawn only on the first recomposition, however it will be called again if this composable goes out of scope. Activity.reportFullyDrawn() has its own check for this and is safe to call multiple times though."

Reply:
Oh yes 😅 . Thanks!